webrtc.html

```diff
@@ -11498,19 +11498,70 @@ <h3 id="mediastreamtrack-network-use">MediaStreamTrack</h3>
+        applicable in WebRTC are defined here. The settings of a track sourced
+        from an <code>RTCPeerConnection</code> represent the currently received
+        frames/samples after downsampling based on applied constraints, if
+        applied. <code>MediaStreamTrack.getCapabilities()</code> MUST always
+        return the empty set.</p>
+        <p>The following constrainable properties are defined to apply only to
+        remote video <code><a>MediaStreamTrack</a></code> objects generated by
+        an <code>RTCPeerConnection</code>:</p>
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Property Name</th>
+              <th>Values</th>
+              <th>Notes</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr id="def-constraint-width">
+              <td><dfn>width</dfn></td>
+              <td><code><a>ConstrainULong</a></code></td>
+              <td>The width or width range, in pixels. The constrainable
+              property allows downscaling. As a setting, this is the width of
+              the latest frame after any downscaling.</td>
+            </tr>
+            <tr id="def-constraint-height">
+              <td><dfn>height</dfn></td>
+              <td><code><a>ConstrainULong</a></code></td>
+              <td>The height or height range, in pixels. The constrainable
+              property allows downscaling. As a setting, this is the height of
+              the latest frame after any downscaling.</td>
+            </tr>
+            <tr id="def-constraint-frameRate">
+              <td><dfn>frameRate</dfn></td>
+              <td><code><a>ConstrainDouble</a></code></td>
+              <td>The exact frame rate (frames per second) or frame rate range.
+              The constrainable property allows downsampling by dropping frames.
+              As a setting, this is an estimate of the frame rate based on
+              recently received frames after any downsampling.</td>
+            </tr>
+            <tr id="def-constraint-aspect">
+              <td><dfn>aspectRatio</dfn></td>
+              <td><code><a>ConstrainDouble</a></code></td>
+              <td>The exact aspect ratio (width in pixels divided by height in
+              pixels, represented as a double rounded to the tenth decimal
+              place) or aspect ratio range. The constrainable property allows
+              downscaling. As a setting, this is the aspect ratio of the latest
+              frame after any downscaling.</td>
+            </tr>
+          </tbody>
+        </table>
+        <p>When downscaling occurs due to applied constraints, the user agent
+        MUST downscale as little as possible while maintaining the original
+        aspect ratio until the target width or height is matched, and then
+        cropping the other dimension if necessary. Scaling occurs around the
+        center of the image.</p>
```

Comment:
I still think we need a normative section here like https://w3c.github.io/mediacapture-screen-share/#downscaling-and-frame-decimation even if it's shorter than what screencapture has.